src/locale/locales/fr/messages.po

```diff
@@ -169,11 +165,17 @@ msgstr "<0/> membres"
 
 #: src/screens/StarterPack/Wizard/index.tsx:485
 msgid "<0>{0} </0>and<1> </1><2>{1} </2>are included in your starter pack"
-msgstr ""
+msgstr "<0>{0} </0>et<1> </1><2>{1} </2>sont inclus dans votre pack de démarrage"
 
 #: src/screens/StarterPack/Wizard/index.tsx:497
-msgid "<0>{0}, </0><1>{1}, </1>and {2} {3, plural, one {other} other {others}} are included in your starter pack"
-msgstr ""
+msgctxt "profiles"
+msgid "<0>{0}, </0><1>{1}, </1>and {2, plural, one {# other} other {# others}} are included in your starter pack"
+msgstr "<0>{0}, </0><1>{1} </1>et {2, plural, one {# autre} other {# autres}} sont inclus dans votre pack de démarrage"
```

Comment:
…have I told you about our lord and saviour _écriture inclusive_? Here, we are talking about people, and they have genders, and having `inclus` is _technically_ correct (because of old rules about "masculine gender wins it all", yeah, it's basically how we learned it in school 😬) but in 2024 we try to do better for obvious reasons :)

We have two options:
- Either we follow the _point médian_ route, and slap a good old `·e·` at the end to create `inclus·e·s`, but we try to minimize the usage of _point médian_ to prevent ~~conservative~~ some people from being noisy about it
- Or we follow the _langage épicène_ route, and try to find an alternative which form doesn't change if the subjects have a non-masculine gender. We can't use `sont présents` (it would be `sont présentes`), but we can say `font partie de`, literally `are part of`. It happens that `partie` here _should never be accorded_, so it's always the same in any genders (or numbers)!

So let's choose this option, and go with `font partie de`.

```suggestion
msgstr "<0>{0}, </0><1>{1} </1>et {2, plural, one {# autre} other {# autres}} font partie de votre pack de démarrage"
```